packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -98,9 +98,11 @@
 
   &__dialog-heading {
     margin: 0;
-    font-weight: 700;
 
     @include bpk-text-base;
+
+    // Temporary override as part of BPK-800; to be refactored after BPK-816 is done
+    font-weight: 700; // stylelint-disable-line order/order
```

Comment:
Are we doing `bpk-text-${textStyle}--bold` or just `bpk-text--bold`? Do we envision having different weights for bold per textStyle?